cogboard-webapp/src/components/FormMessages.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from '@emotion/styled/macro';
+
+
+const FormMessages = ({className, messages, ...others}) => {
+
+  if (messages) {
+    return (
+      <ul className={className} {...others}>
+        {
+          messages.map(message => 
+              <li>{message}</li>)
+        }
+      </ul> 
+    );
+  } else {
+    return null;
+  }
+}
+
+const StyledFormMessages = styled(FormMessages)`
```

Comment:
Move this to styled file in components folder.

Reply:
done